docs/considerations.rst

```diff
@@ -0,0 +1,214 @@
+
+Consider the life of the dataset
+--------------------------------
+If other
+compromises are necessary, consider the life of the data set. If data is being acquired for a discrete
+study that will be acquired, analyzed, and published in a year or two, it might not be worth making
+compromises to acquire multi-echo data. If a data set is expected to be used for future analyses in later
+years, it is likely that more powerful approaches to multi-echo denoising will sufficiently mature and add
+even more value to a data set.
+
+Other multi-echo denoising methods, such as MEICA, the predecessor to tedana, have shown the potential for
+much greater data quality improvements, as well as the ability to more accurately separate visually similar
+signal vs noise, such as scanner based drifts vs slow changes in BOLD signal. These more powerful methods are
+still being improved, and the algorithms are still changing. Users need to have the time and knowledge to look
+at the denoising output from every run to make sure denoising worked as intended. 
+
+You may recover signal in areas affected by dropout
+---------------------------------------------------
+Typical single echo fMRI uses an echo time that is appropriate for signal across most of the brain. While this is effective
+it also leads to drop out in regions with low :math:T_2^* values. This can lead to low or even no signal at all in some areas. 
+If your research question could benefit from having either
+improved signal characteristics in regions such as the orbitofrontal cortex, ventral temporal cortex or 
+the ventral striatum them multi-echo fMRI may be beneficial. 
+
+Consider the cost of added quality control
+------------------------------------------
+If someone wants a push-button
+way to use multi-echo data to improve data quality, that doesn't require as deep an inspection of every output,
+stick with using the weighted average. The developers of tedana look forward to when tedana and other methods
+have sufficiently stable algorithms, which have been validated on a wide range of data sets, so that we can
+recommend the wide use of tedana.
+
+
+
+.. _t2smap: https://tedana.readthedocs.io/en/latest/usage.html#run-t2smap
+
+Acquisition Parameter Recommendations
+=====================================
+There is no empirically tested best parameter set for multi-echo acquisition.
+The guidelines for optimizing parameters are similar to single-echo fMRI.
+For multi-echo fMRI, the same factors that may guide priorities for single echo
+fMRI sequences are also relevant.
+Choose sequence parameters that meet the priorities of a study with regards to spatial resolution,
+spatial coverage, sample rate, signal-to-noise ratio, signal drop-out, distortion, and artifacts.
+
+A minimum of 3 echoes is recommended for running TE-dependent denoising.
+While there are successful studies that don’t follow this rule,
```

Comment:
Which rule, having three echos ? Or earlier and later echos ?

Also, again, maybe Bright method ?